src/components/NotFoundButton.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const NotFoundButton = () => {
+  return (
+    <div className="notfound-404">
+      <h2>You may need to consider going back to our
```

Comment:
Nice design @TheDrizzyWay, but i will suggest the message here should be more precise, an example is specifying or including `we couldn't find this page`,  since we are trying to engage the visitors more. what do you think about this?

Reply:
True. I will ask the rest of the team for suggestions, so we could come up with a better message.